src/pocketmine/permission/BanList.php

```diff
@@ -58,6 +58,23 @@ public function setEnabled(bool $flag){
 		$this->enabled = $flag;
 	}
 
+	/**
+	 * @param string $name
+	 *
+	 * @return BanEntry|null
+	 */
+	public function getEntry(string $name): ?BanEntry{
+		$name = strtolower($name);
+		
+		$this->removeExpired();
+
+		if (!isset($this->list[$name])){
+			return null;
+		}
+
+		return $this->list[$name];
```

Comment:
the `isset()` and this statement could be merged into a single statement using a null coalesce (`$this->list[$name] ?? null`)